pkgs/development/ocaml-modules/csv/csvtool.nix

```diff
@@ -0,0 +1,16 @@
+{ lib, buildDunePackage, ocaml, csv, ocaml_lwt, uutf }:
+
+if !lib.versionAtLeast ocaml.version "4.02"
+then throw "csvtool is not available for OCaml ${ocaml.version}"
+else
+
```

Comment:
It seems that this is not required: you can use the `minimumOCamlVersion` argument to `buildDunePackages` instead.

Reply:
ok, applied